src/@chakra-ui/components/Modal.ts

```diff
@@ -12,57 +12,46 @@ const {
   defaultProps,
 } = modalDefaultTheme
 
-const variantCode = definePartsStyle((props) => ({
-  overlay: {
-    bg: "rgba(0, 0, 0, 0.7)",
-  },
-  dialog: {
-    maxW: "100vw",
-    marginTop: "auto",
-    marginBottom: 0,
-    maxHeight: "50%",
-    borderRadius: 0,
-  },
-  header: {
-    bg: props.colorMode === "dark" ? "rgb(25, 25, 25)" : "rgb(247, 247, 247)",
-    borderColor:
-      props.colorMode == "dark" ? "rgb(242, 242, 242)" : "rgb(51, 51, 51)",
-    borderTop: "1px solid",
-    borderBottom: "1px solid",
-    textTransform: "uppercase",
-    fontWeight: "normal",
-    fontSize: "md",
-    fontFamily: "monospace",
-  },
-  closeButton: {
-    padding: 0,
-    width: "24px",
-    height: "24px",
-    borderRadius: 0,
-    color: "rgb(178, 178, 178)",
-    fontSize: "sm",
-    margin: 0,
-    top: 4,
-    insetInlineEnd: 4,
-    bottom: 4,
-  },
-  body: {
-    padding: 0,
-  },
-}))
-
 export const Modal = defineMultiStyleConfig({
   baseStyle: definePartsStyle((props) =>
     defineMergeStyles(defaultBaseStyle?.(props), {
+      overlay: {
+        bg: "gray.500",
+      },
       dialog: {
-        my: 0,
+        bg: "background.base",
+        gap: "4",
+        padding: "8",
+        maxW: "464px",
+        borderRadius: "md",
```

Comment:
https://www.figma.com/file/NrNxGjBL0Yl1PrNrOT8G2B/ethereum.org-Design-System?type=design&node-id=1630-32303&mode=design

Looks like we're using a 4px border radius, which would be the `base` token:

```suggestion
        borderRadius: "base",
```